featuretools/entityset/serialize.py

```diff
@@ -104,9 +109,11 @@ def write_entity_data(entity, path, format='csv', **kwargs):
         df.to_parquet(file, **kwargs)
     elif format == 'pickle':
         # Dask currently does not support to_pickle
-        if isinstance(df, dd.core.DataFrame):
-            df = df.compute()
-        df.to_pickle(file, **kwargs)
+        if isinstance(df, dd.DataFrame):
+            msg = 'Cannot serialize Dask EntitySet to pickle'
+            raise ValueError(msg)
```

Comment:
We should add a test case for this, but could be done as part of this PR or as part of the other test clean up going on for issue #905